crypto/bio/bio_sock.c

```diff
@@ -352,7 +352,6 @@ int BIO_socket_nbio(int s, int mode)
     int ret = -1;
     int l;
 
-    l = mode;
```

Comment:
because of follwing preprocessor maze, i will not touch this one.

Reply:
Yeah, the same assignment on line 356 could be dropped safely though.

Reply:
All the cases that use `l`, actually set it first, so to me it made sense to remove this one.